README.md

```diff
@@ -167,7 +167,7 @@ peer-reviewed and presented in academic conferences.
 - 🎓 [A Fast and Scalable Payment Network with Bitcoin Duplex Micropayment Channels](https://pdfs.semanticscholar.org/c8d2/b0c1f9b1ca739c340e9dc4a2ff71b5bfbb2e.pdf). Decker C, Wattenhofer R. SSS '15.
 - [The bitcoin lightning network: Scalable off-chain instant payments](https://www.weusecoins.com/assets/pdf/library/Lightning%20Network%20Whitepaper.pdf). Poon J, Dryja T. '15.
 - "Deployable Lightning": [Reaching The Ground With Lightning](https://github.com/ElementsProject/lightning/raw/master/doc/deployable-lightning.pdf). Russell R. '15.
-- 🎓 [Teechan: Payment Channels Using Trusted Execution Environments](https://arxiv.org/pdf/1612.07766). Lind J, Eyal I, Pietzuch P, Sirer EG. FC '17.
+- 🎓 [Teechan: Payment Channels Using Trusted Execution Environments](http://fc17.ifca.ai/bitcoin/papers/bitcoin17-final21.pdf). Lind J, Eyal I, Pietzuch P, Sirer EG. BITCOIN '17.
```

Comment:
Do we consistently use this naming? I haven't been doing that. Should this be "BITCOIN" or "FC BITCOIN"? Also there are papers in the "FC TSCW" which I have just tagged "FC". How should we handle these?

Reply:
We should clarify that it was not part of the main event, otherwise people will think that there's a mistake (since it doesn't appear on the program of FC). I prefer FC BITCOIN.

Reply:
I agree. In that case, could you update the WTSC status of my PoW sidechains paper as well?

Reply:
done

Reply:
Where?

Reply:
now